src/main/java/com/minecrafttas/tasmod/events/client/EventDrawHotbar.java

```diff
@@ -8,6 +8,7 @@ public interface EventDrawHotbar extends EventBase{
 	public void onDrawHotbar();
 	
 	public static void fireOnDrawHotbar() {
+		// No logging, because it is literally rendered every frame... This would spam the console even more
 		for (EventBase eventListener : EventListener.getEventListeners()) {
 			if(eventListener instanceof EventDrawHotbar) {
```

Comment:
do we really need this comment

Reply:
I added a log statement for every event in TASmod, except that one